src/Afsy/Bundle/FrontBundle/Resources/views/Avent/day_2017_22.html.twig

```diff
@@ -0,0 +1,293 @@
+    handlers:
+        main:
+            type: fingers_crossed
+            action_level: debug
+            channels: ["!event"]
+            handler: grouped_main
+            excluded_404s:
+                - ^/
+
+        grouped_main:
+            type: group
+            members: [local_rotated, sentry, graylog]
+
+        local_rotated:
+            level: debug
+            type: stream
+            path: "%kernel.logs_dir%/%kernel.environment%.log"
+
+        sentry:
+            level: error
+            type: raven
+            dsn: "https://5c1493a47fe94474a5f4f808a4bc0b3c:887519852ede415090d7477d48616f73@sentry.io/260357"
+
+        graylog:
+            level: info
+            type: gelf
+            publisher:
+                hostname: 127.0.0.1
+                port: 12201
+
+        console:
+            type:   console
+            process_psr_3_messages: false
+            channels: ["!event", "!doctrine", "!console"]</code></pre>
+
+  <h3 id="lancement-de-notre-appli">Lancement de notre appli</h3>
+  <pre><code class="language-none">./bin/console server:run</code></pre>
+  <p>
+    Une fois l'appli lancée, pointez votre navigateur sur <a href="http://localhost:8000">http://localhost:8000</a>. Vous devriez être accueilli par une page 404. Afin de tester, plusieurs possibilités s'offrent à vous :
+  <p>
+
+  <p>
+    <b>Log classique</b> : votre application utilise déjà le logger ? Vous devriez voir vos lignes arriver dans graylog. Si ces logs sont d'un niveau supérieur ou égal à error, ils seront également envoyés sur Sentry.
+  <p>
+
+  <p>
+    <b>Lancement d'une exception</b> : Editez un controller et lancez une exception. Celle-ci sera transmise à Sentry ainsi que le contexte qu'elle transporte (stacktrace, referrer etc...)
+  <p>
+
+  <h2 id="conclusion">Conclusion</h2>
+
+  <p>
+    Comme nous l'avons vu, quelques lignes de configuration suffisent à avoir un suivi des logs solides pour un début en production.
+    Avec ces outils, Sentry vous tiendra au courant de toutes erreur non catchée tandis que Graylog vous permettra de garder un oeil sur le comportement global de votre application.
+  <p>
+
+	<h2 id="bonnes-pratiques">Bonnes pratiques</h2>
+
+	<h3 id="maitriser-la-taille-des-logs-sur-fichiers">Maitriser la taille des logs sur fichiers</h3>
+	<p>Plutôt que le classique stream, préférer le type rotating_file. Il se chargera tout seul de la rotation des fichiers de log et vous évitera des mauvaises surprises liées à l’oubli de la configuration de logrotage (qui reste le plus efficace dans ce domaine)</p>
+  <pre><code class="language-yaml">local_rotated:
+    type: rotating_file
+    max_files: 15
+    path: "%kernel.logs_dir%/%kernel.environment%.log"
+    level: debug</code></pre>
+
+	<h3 id="ajouter-du-contexte">Ajouter du contexte</h3>
+	<p>Prenons la ligne de log suivannte</p>
+  <pre><code class="language-php">$this->logger->info(sprintf("L'utilisateur N°%b vient de créer la commande N°%b",
```

Comment:
J'aurais plutôt dis `%d` (je ne sais pas si c'est important ou pas)

Reply:
of course.